shimmer/src/main/java/com/cooltechworks/views/shimmer/ShimmerRecyclerView.java

```diff
@@ -80,6 +81,10 @@ private void init(AttributeSet attrs) {
                 setDemoChildCount(a.getInteger(R.styleable.ShimmerRecyclerView_shimmer_demo_child_count, 1));
             }
 
+            if (a.hasValue(R.styleable.ShimmerRecyclerView_shimmer_demo_duration)) {
```

Comment:
Why set the duration value twice? Below you already fetched the value for duration, you do not need to do it twice

Reply:
fixed in 0117999785b0baa102bee00946cd0fc797bd29a9 please have a look at.